easybuild/tools/github.py

```diff
@@ -355,7 +355,8 @@ def fetch_latest_commit_sha(repo, account, branch=None, github_user=None, token=
     return res
 
 
-def download_repo(repo=GITHUB_EASYCONFIGS_REPO, branch=None, account=GITHUB_EB_MAIN, path=None, github_user=None):
+def download_repo(repo=GITHUB_EASYCONFIGS_REPO, branch=None, commit=None, account=GITHUB_EB_MAIN, path=None,
+                  github_user=None):
     """
     Download entire GitHub repo as a tar.gz archive, and extract it into specified path.
     :param repo: repo to download
```

Comment:
As this has a docstring, please add the additional parameter to it.

Reply:
fixed in [b9f6f51](https://github.com/easybuilders/easybuild-framework/pull/4468/commits/b9f6f51e87433694bf76177ccac577eea14604fe)